os/include/tinyara/pm/pm.h

```diff
@@ -244,6 +244,10 @@
 /* Defines max length of device driver name for PM callback. */
 #define MAX_PM_CALLBACK_NAME    32
 
+#define PM_LOCK_PATH					"/proc/power/domains/0/pm_lock"
+#define PM_UNLOCK_PATH					"/proc/power/domains/0/pm_unlock"
+#define PM_SLEEP_PATH					"/proc/power/domains/0/enter_sleep"
```

Comment:
Since we do not allow direct state transition to PM_SLEEP, do we still need this?

Reply:
Will be removed in a later commit as in #6054

Reply:
I didn't see changes made in PR#6054, did you miss out when pushing?

Reply:
Sleep control related functions and macros are deleted as part of #6054 

https://github.com/Samsung/TizenRT/pull/6054/files#diff-22a6c0ef6ce6ad31a503af87b85ec16b0bfe2af93949a4a2d66fda97244ad3ceL573-L575

Reply:
Yes I understand, all functions and macros was removed, so as the path here should also be removed.